README.md

```diff
@@ -0,0 +1,30 @@
+# Potato Farmer
+
+Manages `potato_field` and `potato_plants`. Main way to administrate containers.
+
+## Setup
+
+### Dependencies
+
+There are several dependencies required for the development and deployment of this project.
+The first is Docker Engine and Docker Compose. The following list links to the installation
+instructions for each of these:
+
+* [Docker Engine installation instructions.](https://docs.docker.com/engine/install/)
+* [Docker Compose installation instructions.](https://docs.docker.com/compose/install/linux/)
+
+<!-- TODO: identify and add other dependencies; Python? Rust? -->
+
+### Sub-repository management
+
+The tool `setup.py` can be used to clone (or symlink) project repositories for deployment.
+
+For instance, the following command will clone the `plant_dashboard` and `plant_missions` repositories
```

Comment:
name them as they are named: `potato_plant missions` and `potato_plant_dashboard`. Likewise, indicate these are hosted in the BW organization